services/frontend-service/src/ui/ActionsCart/CheckoutDialog.tsx

```diff
@@ -78,11 +74,34 @@ const ApplyButton = (props: { openNotification: (msg: string) => void; closeDial
     }
 };
 
+const LockMessageInput: VFC<{ setLockMessage: (m: string) => void }> = ({ setLockMessage }) => {
+    const updateInput = useCallback((e) => setLockMessage(e.target.value), [setLockMessage]);
```

Comment:
I think this post might apply here: https://reactjs.org/docs/hooks-faq.html#are-hooks-slow-because-of-creating-functions-in-render

With some more explanation here: https://kentcdodds.com/blog/usememo-and-usecallback
Or maybe this one, too: https://dev.to/milu_franz/demystifying-react-hooks-usecallback-and-usememo-1a8j